patterns/java/src/refactoring_guru/patterns/singleton/example/non_therad_safe/DemoSingleThread.java

```diff
@@ -0,0 +1,11 @@
+package refactoring_guru.patterns.singleton.example.non_therad_safe;
+
+public class DemoSingleThread {
+    public static void main(String[] args) {
+        Singleton singleton = Singleton.getInstance("FOO");
+        Client client = new Client();
+        System.out.println(singleton.name); // FOO
+        client.setNameSingleton("BAR");
+        System.out.println(singleton.name); // FOO
+    }
+}
```

Comment:
Let's put here the same result summary as in DemoMultiThread to make examples uniform. Don't forget to remove comments above.